### PR TITLE
full release of stream restore

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph @czue

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/restore.py [hunk 1]
```diff
@@ -514,8 +514,7 @@ def get_cached_payload(self):
             return CachedResponse(None)
 
         if self.sync_log:
-            stream = STREAM_RESTORE_CACHE.enabled(self.domain)
-            payload = self.sync_log.get_cached_payload(self.version, stream=stream)
+            payload = self.sync_log.get_cached_payload(self.version, stream=True)
```
Comment (czue) [inline corehq/ex-submodules/casexml/apps/phone/restore.py]: is there a `stream=False` branch of code we can delete too? or is that still in use

Comment (snopoke) [inline corehq/ex-submodules/casexml/apps/phone/restore.py]: not really, its just passing the stream param through to `couchdbkit.Document.fetch_attachment`. I think its still useful to have the kwarg in the `get_cached_payload` function for tests / debugging.

**Comment by czue** [comment]
cool, good to merge when tests pass (or now if not relevant)

**Comment by czue** [comment]
kicked build

**Pull request merged**